openshift/dynamic/client.py

```diff
@@ -163,6 +167,7 @@ def delete(self, resource, name=None, namespace=None, label_selector=None, field
 
     @meta_request
     def replace(self, resource, body=None, name=None, namespace=None, **kwargs):
+        body = self.serialize_body(body)
         body = body or {}
```

Comment:
Could this not just be `body = self.serialize_body(body) or {}` ?

Reply:
actually line 171 should have been removed

Reply:
fixed